numbat/modules/physics/temperature_conversion.nbt

```diff
@@ -5,20 +5,24 @@ use units::si
 let _offset_celsius = 273.15
 
 @description("Converts from degree Celsius (°C) to Kelvin.")
+@example("from_celsius(300)", "$300 °C$ in Kelvin.")
```

Comment:
Minor nitpick: Formatting this using LaTeX like this leads to an italic `C` and too little space between the number and the unit (in the examples below). The proper style would use `\text{C}` and use `\,` for whitespace between the number and the unit (unless it's °C/°F). So maybe just:
```suggestion
@example("from_celsius(300)", "300 °C in Kelvin.")
```